wasm_split/Cargo.toml

```
@@ -0,0 +1,8 @@
+[package]
+name = "wasm_split"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-once-cell = { version = "0.5.3", features = ["std"] }
```

Since  PR #3987, we may want to unify all member deps in workspace root for better maintainability.

```suggestion
async-once-cell = { default-features = true, workspace = true, features = ["std"] }
```

In workspace `Cargo.toml`:
```toml
async-once-cell = { default-features = false, version = "0.5.3" }
```